docs/en/stack/ml/df-analytics/dfa-classification.asciidoc

```diff
@@ -65,6 +65,24 @@ have a similar number of data points for each class.
 ////
 
 
+[[dfa-classification-feature-encoding]]
+=== Feature encoding
+
+Every {ml} model can only work with numerical values. For this reason, it is 
+required to transform the categorical values of the relevant features into 
+numerical ones. This process is called feature encoding.
+
+{dfanalytics-cap} automatically performs feature encoding. During which the 
+input data is pre-processed with the following encoding techniques:
+
+* one-hot encoding,
+* target-mean encoding,
+* frequency encoding.
```

Comment:
I think a small description what each of these mean would be good. Or at least links to some nice descriptions.